extended/src/test/java/apoc/util/GoogleCloudStorageContainerExtension.java

```diff
@@ -24,4 +24,8 @@ public GoogleCloudStorageContainerExtension withMountedResourceFile(String resou
         this.withClasspathResourceMapping(resourceFilePath, "/data" + gcsPath, BindMode.READ_ONLY);
         return this;
     }
+
```

Comment:
So che non è strettamente correlato,
ma prova a spostare la classe GoogleCloudStorageContainerExtension in extended-it,
insieme a tutte le classi che vengono richiamate